app/containers/Questions/saga.js

```diff
@@ -144,11 +146,28 @@ export function* getQuestionsWorker({
         followedCommunities,
       );
     }
-
+    const usersSet = new Set();
     questionsList.forEach(question => {
```

Comment:
why not just return new array instead of modifying it?

Reply:
We need a container with unique elements